Framework/Crystal/src/PredictPeaks.cpp

```diff
@@ -125,6 +126,8 @@ void PredictPeaks::init() {
                   "Use an extended detector space (if defined for the"
                   " instrument) to predict peaks which do not fall onto any"
                   "detector. This may produce a very high number of results.");
+  declareProperty("EdgePixels", 0,
```

Comment:
Values less than 0 are simply ignored now. Maybe a validator could be useful here, but depending on how urgent this PR is it can also be added at a later point.